Web/src/components/create-room/CreateRoomPage.tsx

```diff
@@ -35,18 +39,43 @@ const CreateRoomContainer = styled.div`
   }
 `;
 
+const ErrorText = styled.p`
+  color: red;
```

Comment:
```suggestion
  color: var(--red);
```

Reply:
should we rename this to a more general 'error' color?